### ISSUE TITLE
plz help me. 10.0.19041.84

### ISSUE THREAD

**Issue by 1073730081** [stascorp/rdpwrap#1163]
[10.0.19041.84-SLInit]
bInitialized.x86      =CF920
bServerSku.x86        =CF924
lMaxUserSessions.x86  =CF928
bAppServerAllowed.x86 =CF930
bRemoteConnAllowed.x86=CF938
bMultimonAllowed.x86  =CF93C
ulMaxDebugSessions.x86=CF940
bFUSEnabled.x86       =CF944

bInitialized.x64      =103FF8
bServerSku.x64        =103FFC
lMaxUserSessions.x64  =104000
bAppServerAllowed.x64 =104008
bRemoteConnAllowed.x64=104010
bMultimonAllowed.x64  =104014
ulMaxDebugSessions.x64=104018
bFUSEnabled.x64       =10401C

[10.0.19041.84]
LocalOnlyPatch.x86  =1
LocalOnlyOffset.x86 =B46B9
LocalOnlyCode.x86   =jmpshort
SingleUserPatch.x86 =1
SingleUserOffset.x86=3AD27
SingleUserCode.x86  =Zero
DefPolicyPatch.x86  =1
DefPolicyOffset.x86 =3D7D9
DefPolicyCode.x86   =CDefPolicy_Query_eax_ecx
SLInitHook.x86      =1
SLInitOffset.x86    =66658
SLInitFunc.x86      =New_CSLQuery_Initialize

LocalOnlyPatch.x64  =1
LocalOnlyOffset.x64 =87611
LocalOnlyCode.x64   =jmpshort
SingleUserPatch.x64 =1
SingleUserOffset.x64=0BF0C
SingleUserCode.x64  =Zero
DefPolicyPatch.x64  =1
DefPolicyOffset.x64 =17ED5
DefPolicyCode.x64   =CDefPolicy_Query_eax_rcx
SLInitHook.x64      =1
SLInitOffset.x64    =1BDFC
SLInitFunc.x64      =New_CSLQuery_Initialize

![image](https://user-images.githubusercontent.com/33291243/92699015-a9008080-f37f-11ea-8a28-48748935c0fc.png)
![image](https://user-images.githubusercontent.com/33291243/92699309-009eec00-f380-11ea-8058-3543c85b3924.png)
![image](https://user-images.githubusercontent.com/33291243/92699272-f41a9380-f37f-11ea-825b-ea74502ed1ff.png)

This is my termsrv.dll。
[termsrv.dll.txt](https://github.com/stascorp/rdpwrap/files/5200140/termsrv.dll.txt)

**Comment by dimrim** [comment]
I can't help but I can confirm the problem. The same happened on my computer.

**Comment by dimrim** [comment]
Found the solution for 10.0.19041.84!
Thist INI-file don't work :https://raw.githubusercontent.com/asmtron/rdpwrap/master/res/rdpwrap.ini

Take this one: "https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini" !

**Comment by ashkerizan** [comment]
19041.508 anyone?

**Comment by vvbark** [comment]
same problem with 19041.508

**Comment by vvbark** [comment]
Solved problem. 

This ini file is good
> 
> 
> Found the solution for 10.0.19041.84!
> Thist INI-file don't work :https://raw.githubusercontent.com/asmtron/rdpwrap/master/res/rdpwrap.ini
> 
> Take this one: "https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini" !

**Comment by asmtron** [comment]
@dimrim
the entries for 10.0.19041.84 are the same in the two INI files.
The two INI files works on 10.0.19041.84 and 10.0.19041.508.

using [**Autoupdate** ](https://github.com/stascorp/rdpwrap/pull/1160)for **RDP Wrapper** will normally fix this problems automatically.

Autoupdate increase the compatibility by using multiple ini files from **_asmtron, DrDrrae, saurav-biswas_** and **_affinityv_**

**Comment by dimrim** [comment]
Sorry... first, you're doing great thing here. But I didn't invent it and I don't know why:  if you use the one INI-file - everything seems to work, but in fact only one user can log in. If you use the other INI-file, everything works without any problems.
Actualy i don't use the autoupdate. After the first using crashed whole RDP on my windows. Didn't had time to ponder what it was. Maybe it's because of my settings.

**Comment by RostyslavBaidak** [comment]
@Vladimir-hub  thanks!
I'm used Win 10 Pro version 2004 assembly 19041.508
I changed ini file https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini and only after re-installed my RDP Wrapper 1.6.2 and it all worked.

**Comment by gamma-577** [comment]
@dimrim 

Hi! Thank you for the ini file. The offsets for 19041.84 are working flawless on my 32 bit Win 10 Enterprise. I haven't checked for 64 bit version. But they should work fine, I guess.

-Gamma

**Comment by gamma-577** [comment]
> 
> 
> @dimrim
> 
> Hi! Thank you for the ini file. The offsets for 19041.84 are working flawless on my 32 bit Win 10 Enterprise. I haven't checked for 64 bit version. But they should work fine, I guess.
> 
> -Gamma

@dimrim ,

I confirm that the offsets work for my 64 bit win 10 pro as well. Thank you!

-Gamma

**Comment by Holle88** [comment]
> Found the solution for 10.0.19041.84!
> Thist INI-file don't work :https://raw.githubusercontent.com/asmtron/rdpwrap/master/res/rdpwrap.ini
> 
> Take this one: "https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini" !

works for me!

**Comment by MartinRGB** [comment]
> Found the solution for 10.0.19041.84!
> Thist INI-file don't work :https://raw.githubusercontent.com/asmtron/rdpwrap/master/res/rdpwrap.ini
> 
> Take this one: "https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini" !

works for me

**Comment by GWAM** [comment]
> @dimrim
> the entries for 10.0.19041.84 are the same in the two INI files.
> The two INI files works on 10.0.19041.84 and 10.0.19041.508.
> 
> using [**Autoupdate** ](https://github.com/stascorp/rdpwrap/pull/1160)for **RDP Wrapper** will normally fix this problems automatically.
> 
> Autoupdate increase the compatibility by using multiple ini files from **_asmtron, DrDrrae, saurav-biswas_** and **_affinityv_**

Yep, I can confirm this solved my issue too, thanks!

**Comment by 1TheMuffinMan** [comment]
> Solved problem.
> 
> This ini file is good
> 
> > Found the solution for 10.0.19041.84!
> > Thist INI-file don't work :https://raw.githubusercontent.com/asmtron/rdpwrap/master/res/rdpwrap.ini
> > Take this one: "https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini" !

Thank you!!!!!!!!!!!!

**Comment by Eanshine** [comment]
> Found the solution for 10.0.19041.84!
> Thist INI-file don't work :https://raw.githubusercontent.com/asmtron/rdpwrap/master/res/rdpwrap.ini
> 
> Take this one: "https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini" !

wroks for me too! Thank you.

**Comment by sebaxakerhtc** [comment]
Duplicate of https://github.com/stascorp/rdpwrap/issues/1004
